README.md

```diff
@@ -41,6 +41,14 @@ Launch the CDI operator.
 oc create -f https://github.com/kubevirt/containerized-data-importer/releases/download/v1.6.0/cdi-operator.yaml
 ```
 
+Launch the Cluster Network Addons operator.
+```bash
+oc create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/v0.1.0/cluster-network-addons-operator_00_namespace.yaml
+oc create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/v0.1.0/cluster-network-addons-operator_01_crd.yaml
```

Comment:
Shouldn't the HCO Operator itself install these CRDs?

Reply:
I think that it should eventually. However, now it has to be done manually. Correct me if I'm wrong please, I base this only on the readme.

Reply:
That's out of scope for the HCO.  OLM creates the CRDs.